### ISSUE TITLE
Sanity check for classification bounds

### ISSUE THREAD

**Issue by markNZed** [AP6YC/AdaptiveResonance.jl#32]
I made a mistake in setting up the DVFA and did not include the test data when calling data_setup!  The test data included larger values than in the training set. It might be worth adding a check in the classify function to ensure inputs are within the bounds of the min/max that was used during training.

**Comment by AP6YC** [comment]
Good catch! I should have added that type of error checking, but I'll make a new feature and link a PR when I get it fixed. I'll also check that this is the case for other existing modules.
